oakcrime.tf

```diff
@@ -0,0 +1,29 @@
+variable "oakcrime_prod_db_password" {
+  description = "db_password for the production app."
+}
+
+variable "oakcrime_prod_django_secret_key" {
+  description = "django_secret_key for the production app."
+}
```

Comment:
For managing the secret variable values -- should we use a 1password file for that?

Reply:
Suure... I've never shared anything with 1password so I don't know how it works.